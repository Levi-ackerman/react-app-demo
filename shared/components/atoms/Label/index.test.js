import React from 'react'
import { shallow } from 'enzyme'
import Label from '.'
import renderer from 'react-test-renderer'

// const wrap = (props = {}) => shallow(<Label {...props} />)

// it('renders children when passed in', () => {
//   const wrapper = wrap({ children: 'test' })
//   expect(wrapper.contains('test')).toBe(true)
// })

// it('renders props when passed in', () => {
//   const wrapper = wrap({ htmlFor: 'foo' })
//   expect(wrapper.find({ htmlFor: 'foo' })).toHaveLength(1)
// })

describe('Label test', () => {
  it('render correctly', () => {
    const tree = renderer.create(<Label />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
