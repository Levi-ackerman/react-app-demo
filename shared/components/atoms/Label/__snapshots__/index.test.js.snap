// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label test render correctly 1`] = `
<label
  className="Label-bxMjsr gyMNoM"
/>
`;
